ci/run_autoscaling_osp18.yml

```diff
@@ -67,24 +57,21 @@
           tags:
             - setup
 
-        - name: "Run telemetry tests playbook locally on the target host"
-          ansible.builtin.shell:
-            cmd: |
-              ANSIBLE_CONFIG=ci/ansible.cfg ansible-playbook -v  -e @"{{ env_vars_file }}" ci/osp18_functional_tests.yml
-            chdir: "{{ fvt_dir }}"
-          register: output
-
-        - name: Save ansible output to a file
-          ansible.builtin.copy:
-            content: "{{ output.stdout }}"
-            dest: "{{ logs_dir }}/ansible_run.log"
-      always:
+        - name: "Run Telemetry Autoscaling tests"
+          ansible.builtin.import_role:
+            name: telemetry_autoscaling
+          vars:
+            vars_path:  "{{ fvt_dir }}/ci/vars"
```

Comment:
These are already included, IINM, because you used the vars_files option on the playbook.